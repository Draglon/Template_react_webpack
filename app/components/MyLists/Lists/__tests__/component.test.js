import React from 'react';

import ListsComponent from '../component';

describe('<ListsComponent />', () => {
  const props = {
    lists: [
      { id: 1, name: 'List 1', description: 'List 1 - description' },
      { id: 2, name: 'List 2', description: 'List 2 - description' },
    ],
    onConfirm: () => {},
  };

  const component = shallow(<ListsComponent {...props} />);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
