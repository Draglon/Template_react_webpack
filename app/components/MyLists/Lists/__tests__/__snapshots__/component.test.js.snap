// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListsComponent /> should match snapshot 1`] = `
<Row
  gutter={8}
  type="flex"
>
  <Col
    className="lists"
    offset={2}
    span={20}
  >
    <Col
      key="1"
      lg={
        Object {
          "span": 4,
        }
      }
      md={
        Object {
          "span": 6,
        }
      }
      sm={
        Object {
          "span": 8,
        }
      }
      xl={
        Object {
          "span": 4,
        }
      }
      xs={
        Object {
          "span": 12,
        }
      }
    >
      <Card
        actions={
          Array [
            <DeleteItemModalContainer
              onConfirm={[Function]}
              params={
                Object {
                  "listId": 1,
                }
              }
              title="Do you want to delete list?"
              type="delete"
            />,
          ]
        }
        className="top-margin"
        hoverable={true}
      >
        <Link
          replace={false}
          to="list/1"
        >
          <Title
            level={4}
          >
            List 1
          </Title>
          List 1 - description
        </Link>
      </Card>
    </Col>
    <Col
      key="2"
      lg={
        Object {
          "span": 4,
        }
      }
      md={
        Object {
          "span": 6,
        }
      }
      sm={
        Object {
          "span": 8,
        }
      }
      xl={
        Object {
          "span": 4,
        }
      }
      xs={
        Object {
          "span": 12,
        }
      }
    >
      <Card
        actions={
          Array [
            <DeleteItemModalContainer
              onConfirm={[Function]}
              params={
                Object {
                  "listId": 2,
                }
              }
              title="Do you want to delete list?"
              type="delete"
            />,
          ]
        }
        className="top-margin"
        hoverable={true}
      >
        <Link
          replace={false}
          to="list/2"
        >
          <Title
            level={4}
          >
            List 2
          </Title>
          List 2 - description
        </Link>
      </Card>
    </Col>
  </Col>
</Row>
`;
